lib/presentation/profile/widget/profile_picture.dart

```diff
@@ -24,7 +25,7 @@ class ProfilePicture extends StatelessWidget {
       // ignore: unnecessary_cast
       imageProvider = FileImage(_image!) as ImageProvider<Object>;
     } else if (_userId != null) {
-      imageProvider = NetworkImage(
+      imageProvider = CachedNetworkImageProvider(
```

Comment:
I'm not sure if I would cache the user's own profile picture unless we can explicitly invalidate the cache when they upload a new profile image. (This might still not be enough however, since CloudFront may still serve the old version for some time after the image has been updated)

Reply:
I'm restoring all those with cache issues like profile images to use plain `NetworkImage`